tensorflow/core/grappler/optimizers/mkl_remapper_test.cc

```diff
@@ -234,6 +172,8 @@ TEST_F(MklRemapperTest, FuseDepthwiseConv2DWithBiasAndActivation) {
         return ops::Identity(fetch, ops::Relu6(activate, bias_add));
       } else if (activation == "Elu") {
         return ops::Identity(fetch, ops::Elu(activate, bias_add));
+      } else if (activation == "None") {
+        return ops::Identity(s.WithOpName("fetch"), bias_add);
```

Comment:
The `return` outside of if already does this. So I don't think we need this `else if`.